docs-next/pages/tutorials/getting-started-with-create-keystone-next-app/index.mdx

```diff
@@ -0,0 +1,134 @@
+import { Markdown } from '../../../components/Page';
+import Image from 'next/image';
+import Link from 'next/link';
+
+<img
+  src={require('./assets/cover.svg')}
+  alt="A terminal with the output of create-keystone-next-app"
+  style={{
+    margin: '0 -1rem 2rem -1rem',
+    width: 'calc(100% + 2rem)',
+    maxWidth: 'none',
+  }}
+/>
+
+# Getting started with `create-keystone-next-app`
+
+[`create-keystone-next-app`](https://github.com/keystonejs/create-keystone-next-app) is a CLI app that makes it easier for you to get started with Keystone.
+It will generate a couple files for you and install all the dependencies you need to run the AdminUI and start using the GraphQL API.
```

Comment:
```suggestion
It will generate a couple of files for you and install all the dependencies you need to run the Admin UI and start using the GraphQL API.
```